assets/css/tailwind.css

```diff
@@ -31,3 +31,7 @@ button {
   font-size: 14px;
   font-weight: 500;
 }
+
+a {
+  @apply text-blue-200 hover:border-b hover:border-b-blue-200;
+}
```

Comment:
Shouldn't it be covered by `BaseButton`? 🤔

Reply:
Prob batter place for this 👍🏻